### PR TITLE
[FIX] mBarCode undefined in BarCodeReadEvent

### PR DESCRIPTION
**Pull request by seblau**

### CODE REVIEW

File: android/src/main/java/org/reactnative/camera/events/BarCodeReadEvent.java [hunk 1]
```diff
@@ -24,7 +24,7 @@ public static BarCodeReadEvent obtain(int viewTag, Result barCode) {
     if (event == null) {
       event = new BarCodeReadEvent();
     }
-    event.init(viewTag);
+    event.init(viewTag, barCode);
```
Comment (sibelius) [inline android/src/main/java/org/reactnative/camera/events/BarCodeReadEvent.java]: mBarCode?

Comment (seblau) [reply android/src/main/java/org/reactnative/camera/events/BarCodeReadEvent.java]: The parameter name in this method is `barCode`. `mBarCode` gonna bet set in the init method

**Comment by seblau** [comment]
yes, I did. its working

**Comment by sibelius** [comment]
Tks, it is correct

**Pull request merged**